TappyBle/TappyBleCommunicator.swift

```diff
@@ -23,24 +23,26 @@
 
 import Foundation
 import CoreBluetooth
-
+@objc
 class TappyBleCommunicator : NSObject, CBPeripheralDelegate, CBCentralManagerDelegate, TappySerialCommunicator{
 
     private var centralManager : CBCentralManager
     private var tappyPeripheral : CBPeripheral
     private var backingSerialService : CBService?
     private var backingRxCharacteristic : CBCharacteristic?
     private var backingTxCharacteristic : CBCharacteristic?
+    
     public private(set) var state : TappyStatus = TappyStatus.STATUS_CLOSED
     private var tappyName : String
     final var bleDeviceUid : String
+    @objc
     public var error : Error?
     
     
     private var dataReceivedListener : ([UInt8]) -> () = {_ in func emptyDataReceivedListener(data : [UInt8]) -> (){}}
     private var statusListener : (TappyStatus) -> () = {_ in func emptyTappyStatusListener(status : TappyStatus) -> (){}}
     private var packetsToSend: [[UInt8]] = [[UInt8]]()
-    
+    @objc
```

Comment:
Are you certain we need these annotations on private methods too?  You would not be able to call this method (by design) even with the annotation right?